adafruit_lsm303.py

```diff
@@ -237,6 +261,52 @@ def mag_rate(self, value):
         reg_m = ((value & 0x07) << 2) & 0xFF
         self._write_u8(self._mag_device, _REG_MAG_CRA_REG_M, reg_m)
 
+    @property
+    def accel_gain(self):
```

Comment:
Please update this to match the convention above, e.g. `def accel_range(self):`.